cloud/blockstore/libs/daemon/common/ya.make

```diff
@@ -56,4 +56,10 @@ PEERDIR(
     library/cpp/sighandler
 )
 
+IF(NETLINK)
```

Comment:
can you explain how we expected to build blockstore-server with netlink support
is it `ya make -D=NETLINK ...`
maybe add some info to docs
also maybe there is some way to detect that there is libnl in the build environment and then compile with netlink

Reply:
Yep `ya make -DNETLINK` should do it. This was intended as a temporary solution to not break default builds until we make a final decision how to link with libnl

Reply:
As for detection – yes, it provides pkg-config files, but I don't think ya.make gives us enough flexibility to use it. I'll think about how to make process more straightforward

Reply:
I suggest we do this in follow-up PR